layouts/partials/team.html

```diff
@@ -66,7 +66,9 @@ <h2 class="section-heading">{{ with .Site.Params.team.speakers }}{{ . | markdown
             <div class="team-member">
                 {{ $url := printf "img/team/%s" .img }}
                 <img src="{{ $url | absURL }}" class="img-responsive img-circle" alt="{{ .name }}">
-              <h3>{{ .name | markdownify }}</h3>
```

Comment:
could you revert this change please?

Reply:
Okay, just did. But mind that without this change, the CI failed to build when I comment out the default stuff for Speaker2.